UnitTestProject/PostalCodeMaskTest.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using MVC_App.Controllers;
+
+namespace UnitTestProject
+{
+    [TestClass]
+    public class PostalCodeMaskTest
+    {
+        RelationsController relationController = new RelationsController();
+
+        [TestMethod]
+        public void ValueMaskCoincidence()
+        {
+            Assert.AreEqual(relationController.ApplyMask("1488-HH", "NNNN-LL"), "1488-HH");
```

Comment:
Do not write inline.
Use structure AAA:
Arrange
Act
Assert

Also write test cases if (**ApplyMask(string value, string mask)** ) value or mask = String.Empty or value or mask = null.

Reply:
Correct naming:

- TestMaskWithDifferentRegister
- TestMaskWithDifferentCharNumber

and so on